### ISSUE TITLE
🆕 Software Suggestion | 

### ISSUE THREAD

**Issue by replydev** [privacytools/privacytools.io#2026]
## Basic Information

**Name:** Duplicati 2
**Category:** Cloud storage
**URL:** https://www.duplicati.com/  -  https://github.com/duplicati/duplicati

## Description
This software performs client-encrypted backups in many of the existing cloud platforms, without the need to install proprietary clients (like dropbox or mega). 

## Why I am making the suggestion
I believe it is a valid solution to ensure security because the information is encrypted with a key in our possession but also integrity of their data because they reside in remote servers.

## My connection with the software
I am not the developer of Duplicati, I have never contributed to the project but I have been using it for a long time and I am satisfied with it.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by replydev** [comment]
I would add that it has a very good user friendly web interface.
https://www.duplicati.com/screenshots/

**Comment by lynn-stephenson** [comment]
I will look into this more, but from a first impression there does not seem to be any reason to choose this over Cryptomator, nor Kryptor.

It also uses Gnu Privacy Guard, which we are likely going to remove as a recommendation.

**Comment by replydev** [comment]
It has incremental backups feature, compression by default, and it pushes only modified files, saving space and bandwith. It uses privacy guard, but by default AES-256 so i think it's ok. Why GPG is not recommended anymore?

**Comment by AIGeneratedUsername** [comment]
> here does not seem to be any reason to choose this over Cryptomator

@lynn-stephenson Excuse me, but this is like to compare horses and apples. Both tools have different purposes. Cryptomators's main purpose is to make your files encrypted at the cloud.  Duplicati's main purpose is to backup your files.

**Comment by lynn-stephenson** [comment]
Duplicati just encrypts your files (kinda like Cryptomator) before it's sent to a back-up provider (kinda like Cryptomator):

> Amazon S3, Backblaze (B2), Box, Dropbox, FTP, Google Cloud and Drive, HubiC, MEGA, Microsoft Azure and OneDrive, Rackspace Cloud Files, OpenStack Storage (Swift), Sia, Tardigrade, SSH (SFTP), WebDAV, Tencent Cloud Object Storage (COS), and more!

**Comment by lynn-stephenson** [comment]
Duplicati just encrypts your files (kinda like Cryptomator) before it's sent to a back-up provider (kinda like Cryptomator):

> Amazon S3, Backblaze (B2), Box, Dropbox, FTP, Google Cloud and Drive, HubiC, MEGA, Microsoft Azure and OneDrive, Rackspace Cloud Files, OpenStack Storage (Swift), Sia, Tardigrade, SSH (SFTP), WebDAV, Tencent Cloud Object Storage (COS), and more!

**Comment by AIGeneratedUsername** [comment]
> Duplicati just encrypts your files (kinda like Cryptomator) before it's sent to a back-up provider (kinda like Cryptomator):
> 
> > Amazon S3, Backblaze (B2), Box, Dropbox, FTP, Google Cloud and Drive, HubiC, MEGA, Microsoft Azure and OneDrive, Rackspace Cloud Files, OpenStack Storage (Swift), Sia, Tardigrade, SSH (SFTP), WebDAV, Tencent Cloud Object Storage (COS), and more!

Sure, it encrypts the files. But the Duplicati main purpose is to create **automatic** backups to allow a user to restore any old files versions. Cryptomator does not do automatic versioned backups (it does not do automatic backups at all).

**Comment by replydev** [comment]
I agree with @AIGeneratedUsername, as i said in my previous message Duplicati supports **incremental backups** system, like borg, so it can be considered like a cross platform Borg alternative. It doesn't belong to Cryptomator at all.

**Comment by lrq3000** [comment]
This is a very interesting piece of software, and I agree its purpose and functionality is different from cryptomator, as duplicati's main purpose is to be a backup software.

I think it's worth enough that I will make a PR, but I'm not sure where it would fit. It's not a provider so it doesn't belong in the Cloud Storage section (in fact it doesn't provide any storage at all, it manages it, that's different). I guess either Software > File Sync along SyncThing, but the issue is that Duplicati doesn't seem to support syncing across devices, or Software > Self-hosted cloud server, as a Worth Mentioning (there's CryptPad there which IMHO doesn't have anything to do there).

What would fit best?

**Comment by lrq3000** [comment]
Ah I see Cryptomator is listed as a Worth Mentioning under Cloud Provider. If I understand the reasoning behind, then the Worth Mentioning section can be used to list side utilities to the main function of the section, if that's correct then Duplicati can also belong there.

**Comment by AIGeneratedUsername** [comment]
IMHO, it is a lack of "Backup software" section on a website. This software is a versioned backups tool with option to encrypt your data and option to store it somewhere at remote place.

Anyway, listing the software at least somewhere at related topic is a good start point. Maybe it can be refactored if a "Backup software" section on a website will appear. @lrq3000 It would be nice if you will do what you say.

**Comment by lrq3000** [comment]
Yes, and actually I think the best would be to reorganize all the sections on file sharing, file sync, file cloud etc into one page with sections as was done for real-time communications, which would make sense because file management softwares often have multiple purposes and also users don't necessarily know whether they want something to sync files or backup or share, etc.

But we would need a more extensive discussion for such a big change, so for the moment I will just add this app as a Worth Mentioning in a related section.

**Comment by lrq3000** [comment]
PR is ready.
